client/src/components/Cards/Tips/index.js

```diff
@@ -35,24 +37,28 @@ const getColor = (index, startingColor) => {
   return colorArray[_index];
 };
 
-const Tips = ({ tips = [], startingColor = 0, ...props }) => {
+const Tips = ({ tips = [], startingColor = 0, cols, inner, ...props }) => {
   return (
-    <S.Section {...props}>
+    <Row {...props} inner={inner}>
```

Comment:
delete `Section` component from style.js if its not used in somewhere else please

Reply:
done